var/spack/repos/builtin/packages/py-numba/package.py

```diff
@@ -23,8 +23,8 @@ class PyNumba(PythonPackage):
     # That's why it was chosen as an upper bound in the following depends_on
     # calls.  If newer versions maintain backwards compatibility, the calls
     # can be updated accordingly.
-    depends_on('py-llvmlite@0.25:', type=('build', 'run'), when='@0.40.1:')
-    depends_on('py-llvmlite@0.20:0.25', type=('build', 'run'), when='@0.35.1')
+    depends_on('py-llvmlite@0.27.1', type=('build', 'run'), when='@0.40.1:')
```

Comment:
@erimar77 to be clear you want it to depend on exactly `0.27.1` for versions >= `0.40.1` and any version between `0.20.0` and `0.27.1` (inclusive) for earlier versions. To be clear, this will typically mean that all `py-numba` versions will depend on `py-llvmlite@0.27.1`.

Reply:
@erimar77 :point_up: